Source/Core/DolphinQt/Config/Graphics/HacksWidget.cpp

```diff
@@ -276,6 +279,15 @@ void HacksWidget::AddDescriptions()
       "higher internal resolutions. This setting has no effect when native internal "
       "resolution is used.<br><br><dolphin_emphasis>If unsure, leave this "
       "unchecked.</dolphin_emphasis>");
+  static const char TR_NATIVE_TEXTURE_SAMPLING_DESCRIPTION[] = QT_TR_NOOP(
+      "Use the video backend's built-in texture sampling functionality instead of a manual "
+      "implementation.<br><br>This can introduce graphical issues in some games on some GPUs (such "
+      "as breaking pre-rendered VP6 videos in some EA titles), but will generally perform "
+      "better.<br><br>Disabling this option uses manual texture sampling, which more accurately "
+      "handles texture wrapping and can bypass GPU problems.  Some of the accuracy improvements do "
```

Comment:
> Some of the accuracy improvements do not apply when using custom textures or when using a higher internal resolution with Scaled EFB Copy enabled

I'd be curious what accuracy improvements aren't available and whether this new implementation causes issues with higher IR / custom textures..

Reply:
The only accuracy improvement that isn't available is emulating the broken way the console handles wrapping when the texture size isn't a power of 2 (see [bug 12408](https://bugs.dolphin-emu.org/issues/12408)), which is disabled when using custom textures/higher IR it's possible to have textures with a non-power-of-2 size that were originally a power of 2 size.  It's pretty minor and shouldn't affect any official games.  The fixes for bad GPU behavior should still apply.